import_logs.py

```diff
@@ -2098,6 +2123,31 @@ def detect_format(file):
         logging.debug('Format %s is the best match', format.name)
         return format
 
+    def is_filtered(self, hit):
+        host = None
+        if hasattr(hit, 'host'):
+            host = hit.host
+        else:
+            try:
+                host = urlparse.urlparse(hit.path).hostname
+            except:
+                pass
+
+        if host:
+            if config.options.exclude_host and len(config.options.exclude_host) > 0 and host in config.options.exclude_host:
```

Comment:
This is currently an `in`. @mattab I wonder if we need an exact comparison here? or maybe different parameters one for equals and one for contains? In our case, a "contains" could match another host that we do not want to import.

Reply:
‘exclude_host’ is a list.

Reply:
Sorry all good, when I tested it, I looked wrong at the result.